bandit/blacklists/calls.py

```diff
@@ -3,11 +3,8 @@
 # Copyright 2016 Hewlett-Packard Development Company, L.P.
 #
 # SPDX-License-Identifier: Apache-2.0
-
 r"""
-====================================================
-Blacklist various Python calls known to be dangerous
-====================================================
+Blacklist various Python calls known to be dangerous.
```

Comment:
By removing the '===', this changes the font from a heading to normal paragraph text.  However, this text is intended to be a heading.  See https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html

Reply:
Reverting first line missing period warning update.